pkg/redist/System.Collections.Concurrent/pkg/System.Collections.Concurrent.pkgproj

```diff
@@ -0,0 +1,26 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <!-- bump up the version since we are redisting the 4.0.10 bits in a new package -->
```

Comment:
4.0.10 already exists in nuget.org.  So we need to produce a new version.  If we tried to harvest the version we would get 4.0.10 which we can't ship because thats already on nuget